src/js/components/Home/NewPost/AddImage.module.css

```diff
@@ -0,0 +1,25 @@
+.addImage {
+  width: 45px;
+  height: 45px;
+  border-radius: 50%;
+
+  font-size: 35px;
```

Comment:
`35px` to chyba `--l-font-size`, więc można zmienić

Reply:
Tu mamy roziar ikony więc raczej bym tego w ten sposób nie wiązał z rozmiarem czcionki

Reply:
Ok, w takim razie wszystko jest w porządku